### PR TITLE
Dyn 4239 step 4 packages tour

### PR DESCRIPTION
**Pull request by filipeotero**
### Purpose

This PR includes:
- Step 4 in the packages tour
- Fix step numbers 
- Adjust the height of the webbrowser popup

Step4

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang 

### FYIs

@RobertGlobant20

### CODE REVIEW

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -1442,6 +1442,10 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Views\GuidedTour\HtmlPages\Resources\filterIcon.png" />
+    <EmbeddedResource Include="Views\GuidedTour\HtmlPages\Resources\sortIcon.png" />
```
Comment (QilongTang) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:1446]: Hi @filipeotero , would you check if the filter and sort icons are already part of the project?

**Review by QilongTang — COMMENTED** [review]
Looks great witn one question

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj (hunk 1 above)
Comment (filipeotero) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:1446]: Hi @QilongTang, I just found the icons with transparent backgrounds.

Comment (QilongTang) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:1446]: Oh I was not aware that transparent  background not acceptable?

Comment (filipeotero) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:1446]: It's acceptable but I will need to adjust the html haha.
Another thing is that we only search the resources inside _'Views\GuidedTour\HtmlPages\Resources\'_  folder :slightly_frowning_face:

Comment (QilongTang) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:1446]: Make sense, enough difference to keep this version

**Review by QilongTang — APPROVED** [review]

**Pull request merged**